lib/collect.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+"""This is a docstring."""
```

Comment:
What does this module do?  Since this only has the module invocation and no exported API that seems super important unless we chance this to a a proper CLI interface (I'd prefer using click to keep things nice and simple).

Reply:
done.